prow/spyglass/artifacts.go

```diff
@@ -51,6 +51,11 @@ func (s *Spyglass) ListArtifacts(src string) ([]string, error) {
 			break
 		}
 	}
+
+	if err != nil {
+		logrus.Warningf("Failed to list artifacts for prow job: %v", err)
```

Comment:
I think we're doing this on purpose? Perhaps this should be a `Debugf`? `Warning` and `Error` levels are reserved for things that admins should be looking for and fixing and I'm not convinced a normal deployment of `deck` won't just spit this out all the time. It looks like the method is written to handle the log being missing on purpose.